### ISSUE TITLE
Pondering a Slicer crashes the game.

### ISSUE THREAD

**Issue by MagmaSlime123** [PssbleTrngle/SliceAndDice#35]
### What happened?

Title.

### Mod Version

1.1.1

### Minecraft Version

1.18

### Forge Version

40.2.1

### Relevant log output

```shell
N/A. No crash report is made.
```

**Comment by PssbleTrngle** [comment]
Not quite sure if this is the same issue as #37, can you provide the content of the `latest.log` file?

**Comment by MagmaSlime123** [comment]
I looked in the file and I'm not seeing any info about this in there.

**Comment by PssbleTrngle** [comment]
Ok, but I still can't do anything without the information in the log file

**Comment by MagmaSlime123** [comment]
The mod simply crashes with all required mods installed.

Pastebin is either bugging out or is down, so I can't post anything here unless I post the entire report in this comment.

![image](https://user-images.githubusercontent.com/51277766/221249554-c5f2399b-6350-4d76-87cd-c66d3a0b0706.png)

**Comment by gigawhip** [comment]
I had the same experience in v2.0.1. Reverting to v2.0.0 resolved the issue.

**Comment by PssbleTrngle** [comment]
Since I am unable to recreate it, I really need the content of a log file to do anything about it

**Comment by gigawhip** [comment]
The just-released v2.0.3 has resolved the issue for me in Minecraft 1.19.2. Thank you!

The error in v2.0.1 was directly related to the fix in v2.0.3:

```log
Caused by: java.lang.RuntimeException: PartialModel 'sliceanddice:block/slicer/head' loaded after ModelEvent.RegisterAdditional
	at com.jozufozu.flywheel.core.PartialModel.<init>(PartialModel.java:31) ~[flywheel-forge-1.19.2-0.6.8.a.jar%23336!/:0.6.8.a]
	at com.possible_triangle.sliceanddice.SlicerPartials.<clinit>(SlicerPartials.kt:7) ~[sliceanddice-2.0.1.jar%23388!/:2.0.1]
```
